indexScratch.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Parallax Slider with Form</title>
+   <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div id="slider" class="swipe">
+        <div class="swipe-wrap">
+
+            <!--START COMMENTED OUT TRAINING SLIDES-->
```

Comment:
Commented out text will be removed after reviewed with instructor.

Reply:
Great, looks fine for a test of the Pull Request functionality.